content/en/monitors/monitor_types/metric.md

```diff
@@ -79,10 +79,12 @@ further_reading:
 
 **Note:** No Data Alerts have a default max of 24 hours. Reach out to support to discuss increasing this value.
 
-7. Opt to **automatically resolve the monitor from a triggered state**.
-    In general you'll want to leave this option off as you only want an alert to be resolved when it's fixed.
+7. Opt to **automatically resolve the monitor from a triggered state**. By enabling this option, the monitor will resolve after a certain amount of time rather than meeting its defined recovery threshold. In general you would want to leave this option off as you only want an alert to be resolved when it's fixed.
```

Comment:
```suggestion
7. Opt to **automatically resolve the monitor from a triggered state**. By enabling this option, the monitor resolves after a certain amount of time rather than meeting its defined recovery threshold. In general, you want to leave this option off so the alert only resolves when it's fixed.
```